tests/core/test_tracker_stores.py

```diff
@@ -123,16 +123,7 @@ def test_find_tracker_store(default_domain: Domain, monkeypatch: MonkeyPatch):
 
 
 class ExampleTrackerStore(RedisTrackerStore):
-    def __init__(self, domain, url, port, db, password, record_exp, event_broker=None):
-        super().__init__(
-            domain,
-            event_broker=event_broker,
-            host=url,
```

Comment:
can remove it all now that it is `host` so the `__init__` can be pulled directly from RedisTrackerStore class